### ISSUE TITLE
bug: `materialyou` patch is broken in latest patches

### ISSUE THREAD

**Issue by kapchapka** [inotia00/ReVanced_Extended#266]
### Type

Cosmetic

### Application

Youtube 18.06.35

### Bug description

Material you thene doesn't work properly with the latest patches version(2.162.1)

### Steps to reproduce

Step 1. Including ″material you″ patch.
Step 2. Choosing ″device settings″ in appearance.

### Relevant log output

```shell
...
```

### Device Environment

Android 13, OneUI 5

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by kazimmt** [comment]
```        <item name="colorOnPrimaryFixed">@color/gm3_sys_color_on_primary_fixed</item>
        <item name="colorOnPrimaryFixedVariant">@color/gm3_sys_color_on_primary_fixed_variant</item>
        <item name="colorOnSecondaryFixed">@color/gm3_sys_color_on_secondary_fixed</item>
        <item name="colorOnSecondaryFixedVariant">@color/gm3_sys_color_on_secondary_fixed_variant</item>
        <item name="colorOnTertiaryFixed">@color/gm3_sys_color_on_tertiary_fixed</item>
        <item name="colorOnTertiaryFixedVariant">@color/gm3_sys_color_on_tertiary_fixed_variant</item>
        <item name="colorPrimaryFixed">@color/gm3_sys_color_primary_fixed</item>
        <item name="colorPrimaryFixedDim">@color/gm3_sys_color_primary_fixed_dim</item>
        <item name="colorSecondaryFixed">@color/gm3_sys_color_secondary_fixed</item>
        <item name="colorSecondaryFixedDim">@color/gm3_sys_color_secondary_fixed_dim</item>
        <item name="colorTertiaryFixed">@color/gm3_sys_color_tertiary_fixed</item>
        <item name="colorTertiaryFixedDim">@color/gm3_sys_color_tertiary_fixed_dim</item>```
        
New strings found inside values-v31-night style.xml 
maybe that's the reason.

**Comment by inotia00** [comment]
fixed in [revanced-patches-v2.162.6](https://github.com/inotia00/revanced-patches/releases/tag/v2.162.6)
